### ISSUE TITLE
[Feature request]Simhub to forward telemetry received to Simhub running on a remote computer

### ISSUE THREAD

**Issue by RangeyRover** [SHWotever/SimHub#1411]
Simhub can happily forward on UDP telemetry that it receives on an alternative port, this already is a feature.

I'm trying to create a situation where simhub can be used to monitor a game running on a remote PC, in particular for the creating of broadcast overlays with the data in AMS2

**Describe the solution you'd like**
Simhub on another remote PC to be able to be configured to receive this remote UDP stream, and also, if possible, to retransmit received telemetry when the source is not a UDP stream.
The ability to manually configure simhub to monitor for a remote network stream.

**Describe alternatives you've considered**
At the moment, I gather telemetry I'm interested in, put them in a dash, then use a browser to collect that data from a dash. it's a very clunky and inefficient workaround.

**Comment by SHWotever** [comment]
Hi ! Unfortunately one of the simhub pillar is to expose everything available from a game, one most games that represents a quite compact dataset but on some others ((RF2, AMS2, iracing ...) that can represent represent up to a good 8MB per second when serialized which is huge. I already experimented ways to do it years ago but this has been leading to a dead end, the network consumption was not reasonable.

Alternatively you can setup a VPN (don't expose simhub on the internet !) and access remotely dashboards using a simple browser.

**Comment by RangeyRover** [comment]
Hi, thanks for the response. I actually did start to receive data on the remote computer (Note remote being on the same internal network)  at 888Kpbs. Most of the dashes worked, the leaderboard was a little weird it didnt get best lap data, but other than that it definitely works, if the game you are using forwards UDP. 
For me, it would be better to encapsulate the telemetry in the form Simhub receives it, usually from shared memory as that seems to be a richer source of information. GarySwallow plugin does not seem to work at all in AMS2 in UDP telemetry. (I know that's a 3rd party plugin and not simhub's deal)
In summary, what I asked for actually works, if it's a UDP based game.
Maybe the better request would be for simhub to retransmit to another (on same internal network) copy of simhub, in my use case, for the purpose of broadcast overlay.
